doc/website/advanced/configuration-guide.md

```diff
@@ -0,0 +1,150 @@
+# Configuration guide
+
+## CMake switches for configuring iceoryx_posh build
+
+When building iceoryx_posh, there are several configuration options set by default.
+These options adjust the limits of Publisher and Subscriber Ports for the resource management. These limits are used to create management structures in the shared memory segment called `iceoryx_mgmt` when starting up RouDi.
+
+ |  switch  |  description |
+ |:---------|:-------------|
+ | `IOX_MAX_PUBLISHERS` | Maximum number of publishers which can be managed by one `RouDi` instance |
+ | `IOX_MAX_SUBSCRIBERS_PER_PUBLISHER` | Maximum number of connected subscriber ports per publisher port |
+ | `IOX_MAX_PUBLISHER_HISTORY` | Maximum number of chunks available for the publisher history |
+ | `IOX_MAX_CHUNKS_ALLOCATED_PER_PUBLISHER_SIMULTANEOUSLY` | Maximum number of chunks a publisher can allocate at a given time |
+ | `IOX_MAX_SUBSCRIBERS` | Maximum number of subscribers which can be managed by one `RouDi` instance |
+ | `IOX_MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY` | Maximum number of chunks a subscriber can hold at a given time (subscriber history size)|
+ | `IOX_MAX_INTERFACE_NUMBER` | Maximum number of interface ports which are used for gateways |
+
+Have a look at [iceoryx_posh_deployment.cmake](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_posh/cmake/iceoryx_posh_deployment.cmake) for the default values of the constants.
+
+!!! hint
+    With the default values set, the size of `iceoryx_mgmt` is ~64.5 MByte. You can reduce the size by decreasing the values from the table via the CMake options. The current values are printed in the CMake stage when building iceoryx.
+
+Example:
+
+```bash
+cmake -Bbuild -Hiceoryx_meta -DIOX_MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY=64
+```
+
+With that change the footprint of the management segment is reduced to ~52.7 MBytes. For larger use cases you can increase the value to avoid that samples are dropped on the subscriber side (see also [#615](https://github.com/eclipse-iceoryx/iceoryx/issues/615)).
+
+## Configuring Mempools for RouDi
```

Comment:
I think https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/conceptual-guide.md#organization should provide a link to this chapter. The current link there seems to be wrong. Can you please change it there?

Reply:
link added